integration/docker-compose.pxe.yml

```diff
@@ -5,31 +5,42 @@ services:
   pxe:
     build:
       context: pxe
-    ports:
-      - 8082:8082
     environment:
       NODE_IP_SUBNET: 10.127.127.0
       NODE_IP_NETMASK: 255.255.255.0
       NODE_IP_RANGE_MIN: 10.127.127.100
       NODE_IP_RANGE_MAX: 10.127.127.253
       NODE_IP_ADDRESS: 10.127.127.3
     volumes:
-      - ./pxe/grubx64.efi:/var/lib/tftpboot/grubx64.efi:ro
       - ./pxe/dhcpd.conf.template:/etc/dhcp/dhcpd.conf.template:ro
     networks:
       opi:
         ipv4_address: 10.127.127.3
     healthcheck:
-      test: curl --silent --fail http://localhost:8082 || exit 1
+      test: tftp localhost -v -c get grubx64.efi || exit 1
       interval: 6s
```

Comment:
It's not a huge deal, but downloading the 1MB file every 6s seems a bit excessive.  Maybe increase the interval?

Reply:
yep, all of our healthchecks are terrible... just copy-pasted them... I will open an issue on that ! thanks